src/main/java/graphql/execution/ExecutionStrategyParameters.java

```diff
@@ -50,35 +50,37 @@ public NonNullableFieldValidator nonNullFieldValidator() {
     }
 
     public ExecutionStrategyParameters transform(Consumer<Builder> builderConsumer) {
-        Builder builder = newParameters(this);
+        Builder builder = new Builder(this);
         builderConsumer.accept(builder);
         return builder.build();
     }
 
-    public static Builder newParameters() {
-        return new Builder();
-    }
-
-    public static Builder newParameters(ExecutionStrategyParameters oldParameters) {
```

Comment:
yes, you are right: it would be nicer to just call the copy constructor directly and remove this method.